### ISSUE TITLE
Game crashed the moment an ice dragon started attacking me [game version 1.16.5 - Ice&Fire version 2.1.8]

### ISSUE THREAD

**Issue by YeetyGodofWalls** [Alex-the-666/Ice_and_Fire#4171]
I'm playing on a single-player world with about 40 mods.
I played for about 15 minutes on that world when suddenly I see a flying ice dragon (It was either level 3 or level 4, I don't really know the difference) near a village with a Mausoleum, stupidly I decided to approach the ice dragon with my enchanted copper sword, shield, and full iron armor, I tried defending myself with the shield against the dragon's ice breath, I started getting damaged, and the game froze for about one second before crashing, there is no crash report and the launcher gave me the exit code: 0 error that usually happens when a modded game crashes.
I don't know if it happened because of a mod conflict or if it's something with the mod itself so I'll add my mod list just in case.

**To Reproduce**
1. Go to an ice dragon with a shield
2. Defend yourself with the shield
3. Game gets crashed

**Expected behavior**
I expected to die to the ice dragon after a short, normal fight.
![image](https://user-images.githubusercontent.com/89551394/130868217-9b268bbd-91b9-400e-a005-ac4d1c537344.png)

**Comment by LordBoos** [comment]
This happens to me too. Using server version and client version. Server crashes in the moment when Ice dragons "Ice ball" hits the ground (hit was near my location).
```
[22:34:46] [Server thread/FATAL] [ne.mi.co.ForgeMod/]: Preparing crash report with UUID 0c51ffad-6491-422c-91b4-3d70023f169b
[22:34:46] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
net.minecraft.crash.ReportedException: Ticking entity
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:855) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:paxi.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:291) ~[?:?] {re:classloading,pl:accesstransformer:B}
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:787) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:paxi.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:paxi.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:paxi.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_301] {}
Caused by: java.lang.NullPointerException
        at com.github.alexthe666.citadel.server.entity.datatracker.EntityProperties.getEntity(EntityProperties.java:63) ~[citadel:1.8.0-1.16.5] {re:classloading}
        at com.github.alexthe666.iceandfire.entity.props.FrozenEntityProperties.setFrozenFor(FrozenEntityProperties.java:48) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
        at com.github.alexthe666.iceandfire.entity.IafDragonDestructionManager.lambda$destroyAreaIceCharge$15(IafDragonDestructionManager.java:342) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384) ~[?:1.8.0_301] {}
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:580) ~[?:1.8.0_301] {}
        at com.github.alexthe666.iceandfire.entity.IafDragonDestructionManager.destroyAreaIceCharge(IafDragonDestructionManager.java:337) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
        at com.github.alexthe666.iceandfire.entity.EntityDragonIceCharge.func_70227_a(EntityDragonIceCharge.java:174) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
        at com.github.alexthe666.iceandfire.entity.EntityDragonIceCharge.func_70071_h_(EntityDragonIceCharge.java:79) ~[iceandfire:2.1.8-1.16.5] {re:classloading}
        at net.minecraft.world.server.ServerWorld.func_217479_a(ServerWorld.java:611) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architects_palette.mixins.json:ServerWorldMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:charm.mixins.json:ServerWorldMixin,pl:mixin:APP:charm.mixins.json:accessor.ServerWorldAccessor,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.world.World.func_217390_a(World.java:554) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:twilightforest:hitbox,pl:mixin:APP:sereneseasons.mixins.json:MixinWorld,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:404) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architects_palette.mixins.json:ServerWorldMixin,pl:mixin:APP:abnormals_core.mixins.json:ServerWorldMixin,pl:mixin:APP:endergetic.mixins.json:ServerWorldMixin,pl:mixin:APP:quark.mixins.json:ServerWorldMixin,pl:mixin:APP:charm.mixins.json:ServerWorldMixin,pl:mixin:APP:charm.mixins.json:accessor.ServerWorldAccessor,pl:mixin:APP:enhancedcelestials.mixins.json:MixinServerWorld,pl:mixin:APP:cavesandcliffs.mixins.json:common.world.gen.ServerWorldMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:851) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:betterendforge.mixins.json:MinecraftServerMixin,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:paxi.mixins.json:MixinMinecraftServer,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:A,pl:runtimedistcleaner:A}
        ... 5 more
```

**Comment by AlexModGuy** [comment]
fixed in 2.1.9
